tasks/update-infra-deployments.yaml

```diff
@@ -47,7 +56,7 @@ spec:
       image: quay.io/chmouel/github-app-token@sha256:bc45937ae588df876555ebb56c36350ed74592c7f55f2df255cabef39c926a88
       env:
         - name: GITHUBAPP_KEY_PATH
-          value: $(workspaces.deploy-key.path)/appstudio-staging-ci
+          value: $(tmp/infra-deployments-pr-creator)/appstudio-staging-ci
```

Comment:
should be path directly (not evaluated as `$()`) -> `value: /secrets/deploy-key/appstudio-staging-ci`